const/styles/global.ts

```diff
@@ -153,10 +153,10 @@ const GlobalStyles = createGlobalStyle`
   }
 
   .mobileOnly {
-    display: none;
+    display: none !important;
     
     ${Media.mobile} {
-      display: block;
+      display: block !important;
```

Comment:
I don't know CSS I just know this is not good 😬

Reply:
Because we need this class to override other `display` value which I think in this case is intended to do that.

Reply:
Not a big deal, just being an ass =P

Reply:
I'll tag @michelbio just so he can review when he is back :)

Reply:
Just reviewed: @nenadV91 comments is right in that the `.mobileOnly` was meant as a 'last resort' utility class (same as we used to have `.hide` to override anything using `!important`. In this case the `!important` could be justified, albeit not pretty or best practice.

No direct conflicts, but only because our code base is so small. Could cause future issues and would opt trying to get rid of it's usage, in favor of passing props.

Can leave for now, as this currently only affects 1 button, under the 'How it works'. The idea was to only show this button on mobile. Could have been a passed prop instead to make it cleaner, as mentioned above.

I most definitely will cleanup/refine when I'm back!

Reply:
Just reviewed: @nenadV91 comments is right in that the `.mobileOnly` was meant as a 'last resort' utility class (same as we used to have `.hide` to override anything using `!important`. In this case the `!important` could be justified, albeit not pretty or best practice.

No direct conflicts, but only because our code base is so small. Could cause future issues and would opt trying to get rid of it's usage, in favor of passing props.

Can leave for now, as this currently only affects 1 button, under the 'How it works'. The idea was to only show this button on mobile. Could have been a passed prop instead to make it cleaner, as mentioned above.

I most definitely will cleanup/refine when I'm back!